### DISCUSSION TITLE
Publications using metagenome-atlas

### DISCUSSION THREAD

**Discussion by SilasK** [metagenome-atlas/atlas#421] [Show and tell]
If you used metagenome-atlas for your publication add them to this "Hall of Fame" 🥇 .
Add the DOI to your publication and maybe explain what you used atlas for. You can also link to code you want others to use.

**Comment by SilasK** [comment]
Ok. I go first. 

Title: **Warmth Prevents Bone Loss Through the Gut Microbiota**

https://doi.org/10.1016/j.cmet.2020.08.012


The best project I used atlas for was for analyzing mice gut metagenomes of warm exposed mice. My colleague Claire Chevalier found out that warm exposure of mice leads to better bones. It also changes the gut microbiome and transplanting of the microbiome of warm-adapted mice can improve the bone strength in recipients. 

Now the cool metagenomic part was: Assembling MAGs with atlas. Then we predicted functional pathways with @LeeBergstrand 's  pygenomeprop. We found predicted increased Polyamine synthesis in the microbiome of warm exposed mice. This change could be confirmed with targeted metabolomics. This genome-centered approach allows also us to identify the driver species of the change in the pathways. 

![Warm_finctional](https://user-images.githubusercontent.com/7523034/129746803-42cc0b17-6af4-4129-8566-58c48ea29747.jpg)

While the pathway prediction was performed outside of Atlas We are looking to implement a KEGG pathway prediction inside atlas See #360

**Comment by jmtsuji** [comment]
Along with the rest of our team, @espasov and I used ATLAS to explore ammonia-oxidizing microbial communities in a wastewater treatment plant (WWTP).

## **Title:** High functional diversity among _Nitrospira_ populations that dominate rotating biological contactor microbial communities in a municipal wastewater treatment plant
Emilie Spasov, Jackson M. Tsuji, Laura A. Hug, Andrew C. Doxey, Laura A. Sauder, Wayne J. Parker & Josh D. Neufeld
ISME J, 2020

https://doi.org/10.1038/s41396-020-0650-2

@espasov wrote a great summary article about this work at [this link](https://naturemicrobiologycommunity.nature.com/posts/we-were-missing-a-key-piece-all-along-comammox-nitrospira-dominate-a-wastewater-community). In short, we combined metagenomics, ammonia/nitrite/nitrate measurements, and other molecular methods with new knowledge of the complete ammonia oxidation (comammox) process to show that comammox bacteria likely play an important role in wastewater treatment at this WWTP. Our discovery revises previous assumptions about the key players in ammonia oxidation in this WWTP, and our work broadens the known global distribution/ecology of comammox bacteria.

The exciting discovery facilitated by ATLAS was that potential complete ammonia-oxidizing (comammox) bacteria in the _Nitrospira_ genus were highly diverse at the population level. We were able to reconstruct ~12 high completness & low contamination metagenome-assembled genomes (MAGs) associated with the _Nitrospira_, and many of these bins appeared to have genomic capacity for complete ammonia oxidation. Excitingly, we also found that some potential commamox _Nitrospira_ bins encoded genes for cyanate metabolism, a trait that had not been associated with comammox bacteria previously.

![Comammox _Nitrospira_ metabolic potential](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41396-020-0650-2/MediaObjects/41396_2020_650_Fig7_HTML.png?as=webp)

After getting the dereplicated MAGs from ATLAS, the gene heatmap above was generated using [BackBLAST](https://github.com/LeeBergstrand/BackBLAST_Reciprocal_BLAST), a pipeline that @LeeBergstrand and I are working on (currently in alpha release of v2).

**Reply by SilasK** [reply]
Very cool!

**Comment by CedricMidoux** [comment]
I add my little contribution

# Mining the equine gut metagenome: poorly-characterized taxa associated with cardiovascular fitness in endurance athletes

Mach, N., Midoux, C., Leclercq, S. et al. 
Commun Biol 5, 1032 (2022).
https://doi.org/10.1038/s42003-022-03977-7

![](https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs42003-022-03977-7/MediaObjects/42003_2022_3977_Fig2_HTML.png)

We used ATLAS to build a MAG catalogue and genes to identify an associative link between endurance performance and gut microbiome composition for elite endurance horses.
